devbox/src/syncer/SkipScanActor.scala

```diff
@@ -112,6 +113,12 @@ class SkipScanActor(sigActor: castor.Actor[SigActor.Msg],
               PathMap.from(
                 paths
                   .walkSubPaths(src.segments)
+                  .filter(subsegments =>
+                    syncIgnoreRegex match {
+                      case None => true
+                      case Some(r) => !r.matches(subsegments.mkString("/"))
```

Comment:
We probably don't care, but we might use `File.separator` to be somewhat more cross-platform

Reply:
Yeah we could, but I think we don't work on windows right now anyway due to shelling out to `ssh` and stuff